src/nek5000_driver.cpp

```diff
@@ -143,6 +155,10 @@ int Nek5000Driver::set_heat_source_at(int32_t local_elem, double heat)
   return nek_set_heat_source(local_elem + 1, heat);
 }
 
+void Nek5000Driver::write_step(int timestep, int iteration) {
+  nek_write_step();
```

Comment:
This is implemented here:
https://github.com/enrico-dev/Nek5000/blob/e7edb50a18b272015bec9ea8946267ce69e85b47/core/nek_interface.f#L268-L279